tests/sentry/workflow_engine/migration_helpers/test_migrate_rule.py

```diff
@@ -187,3 +188,31 @@ def test_required_fields_only(self):
         assert if_dcg.logic_type == DataConditionGroup.Type.ANY_SHORT_CIRCUIT
         filters = DataCondition.objects.filter(condition_group=if_dcg)
         assert filters.count() == 0
+
+    def test_delete_issue_alert(self):
+        # TODO: update after action registry is merged
+        pass
+
+    def test_delete_issue_alert_no_actions(self):
+        migrate_issue_alert(self.issue_alert, self.rpc_user)
+
+        alert_rule_workflow = AlertRuleWorkflow.objects.get(rule=self.issue_alert)
+        workflow = alert_rule_workflow.workflow
+        when_dcg = workflow.when_condition_group
+        if_dcg = WorkflowDataConditionGroup.objects.get(workflow=workflow).condition_group
+
+        assert when_dcg is not None
+        assert if_dcg is not None
+
+        conditions = DataCondition.objects.filter(condition_group=when_dcg)
+        assert conditions.count() == 2
+        filters = DataCondition.objects.filter(condition_group=if_dcg)
+        assert filters.count() == 1
+
+        delete_migrated_issue_alert(self.issue_alert)
```

Comment:
does the `AlertRuleWorkflow` object also get deleted?

Reply:
good point, forgot about that